docs/contributing.md

```diff
@@ -66,7 +66,7 @@ is stored as the `NPM_TOKEN` secret on GitHub Actions.
 
 Docs are automatically published using [Read The Docs](https://readthedocs.org/) upon a new Release.
 
-You can also publish the latest release of the docs by running the [Release Latest Documentation](https://github.com/rjsf-team/react-jsonschema-form/actions/workflows/release.yml) workflow.
+You can also publish the latest release of the docs by running the [Release Latest Documentation](https://github.com/rjsf-team/react-jsonschema-form/actions/workflows/release-docs.yml) workflow.
```

Comment:
We should link to https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow.